kalliope/stt/cmusphinx/README.md

```diff
@@ -0,0 +1,24 @@
+### CMU Sphinx
+
+The CMU Sphinx STT is based on the [CMUSPhinx engine](http://cmusphinx.sourceforge.net/wiki/).
+Only english language is available.
+
+#### Installation
+
+Install package
+```bash
+sudo apt-get install python python-all-dev python-pip build-essential swig git libpulse-dev
```

Comment:
Most of the packages are already provided by Kalliope install. 
Don't we have a risk of conflict ? like python-pip could have been installed differently